### PR TITLE
SPUI-1274 > Add fee in transaction

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/service/services/operation_history/model/operation_history_response_model.dart [hunk 1]
```diff
@@ -105,10 +105,13 @@ class SwapInfo with _$SwapInfo {
     required bool isSell,
     required String sellAssetId,
     required String buyAssetId,
+    required String feeAsset,
     @DecimalSerialiser() required Decimal sellAmount,
     @DecimalSerialiser() required Decimal buyAmount,
     @DecimalSerialiser() required Decimal baseRate,
     @DecimalSerialiser() required Decimal quoteRate,
+    @DecimalSerialiser() required Decimal feeAmount,
+    @DecimalSerialiser() required Decimal feePerc,
```
Comment (eli1stark) [inline lib/service/services/operation_history/model/operation_history_response_model.dart:114]: rename to feePercent, use @JsonKey(name: 'feePerc')

File: lib/service/services/swap/model/get_quote/get_quote_response_model.dart [hunk 2]
```diff
@@ -21,6 +22,8 @@ class GetQuoteResponseModel with _$GetQuoteResponseModel {
     @JsonKey(name: 'toAssetVolume')
         required Decimal toAssetAmount,
     @JsonKey(name: 'actualTimeInSecond') required int expirationTime,
+    @DecimalSerialiser() required Decimal feeAmount,
+    @DecimalSerialiser() required Decimal feePercentage,
```
Comment (eli1stark) [inline lib/service/services/swap/model/get_quote/get_quote_response_model.dart:26]: rename to feePercent, use @JsonKey(name: 'feePercentage')

File: lib/service/services/operation_history/model/operation_history_response_model.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/service/services/operation_history/model/operation_history_response_model.dart]: done

File: lib/service/services/swap/model/get_quote/get_quote_response_model.dart (hunk 2 above)
Comment (DenisMartych) [reply lib/service/services/swap/model/get_quote/get_quote_response_model.dart]: done

**Pull request merged**